x-pack/plugins/infra/public/components/range_date_picker/index.tsx

```diff
@@ -255,6 +255,7 @@ export const RangeDatePicker = injectI18n(
           id="QuickSelectPopover"
           button={quickSelectButton}
           isOpen={this.state.isPopoverOpen}
+          // @ts-ignore
```

Comment:
here callback is defined as `private closePopover = (type: string, from?: string, to?: string) => {`
but `closePopover` in `<EuiPopover` doesn't pass any arguments. tried to fix it with making all params optional
```js
private closePopover = (type?: string, from?: string, to?: string) => {
```
but it leads to other errors in this file, so suppressed for now